app/src/main/java/org/mozilla/fenix/library/bookmarks/edit/EditBookmarkFragment.kt

```diff
@@ -135,6 +136,7 @@ class EditBookmarkFragment : Fragment(R.layout.fragment_edit_bookmark) {
 
     private fun initToolbar() {
         val activity = activity as? AppCompatActivity
+        (activity as? HomeActivity)?.getSupportActionBarAndInflateIfNecessary()
         val toolbar = activity?.findViewById<Toolbar>(R.id.navigationToolbar)
```

Comment:
We can just get `toolbar` from `getSupportActionBar...`, right?

I'm tempted to say we should rename `getSupportActionBar...` as `getToolbar` but then it won't be as discoverable. Maybe we should comment as much in the method kdoc. :)